ggshield/verticals/secret/output/secret_text_output_handler.py

```diff
@@ -240,46 +200,65 @@ def leak_message_located(
                         max_width,
                     )
                     leak_msg.write(formatted_line)
+                else:
+                    # The secret is on multiple lines
+                    detector_position = float("inf"), float("-inf")
+                    for index, line_of_secret in enumerate(match.lines_with_secret):
```

Comment:
similar here: `line_of_secret` → `line`

Reply:
the problem here is that we already have a line define :)

Reply:
Oh, I missed that. Might be because that function is quite large (8 level of indentations at its peak). I won't object if you split it 😉, but not a blocker if you want to land this first.